build.gradle

```diff
@@ -1,46 +1,81 @@
 plugins {
-    id "com.jfrog.bintray" version "1.7"
-    id "com.github.jruby-gradle.base" version "0.1.5"
     id "java"
+    id "maven-publish"
+    id "org.embulk.embulk-plugins" version "0.2.6"
     id "checkstyle"
     id "findbugs"
     id "jacoco"
     id "com.github.kt3k.coveralls" version "2.4.0"
 }
-import com.github.jrubygradle.JRubyExec
 
 repositories {
     mavenCentral()
     jcenter()
     maven { url "https://dl.bintray.com/embulk-input-jdbc/maven" }  // for embulk-input-jdbc
 }
-configurations {
-    provided
-}
 
-version = "0.2.0"
+// TODO: Configure its Maven package group.
+// group = ""
+version = "0.3.0"
+description = "Translate schema and column values by SQL-like query provided by Apache Calcite."
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.8.26"
-    provided "org.embulk:embulk-core:0.8.26"
-    compile  "org.embulk.input.jdbc:embulk-input-jdbc:0.8.2"
-    compile  "org.apache.calcite:calcite-core:1.13.0"
+    compileOnly "org.embulk:embulk-core:0.9.23"
+    compile("org.embulk.input.jdbc:embulk-input-jdbc:0.8.2") {
+        exclude group: "org.embulk", module: "embulk-core"
+    }
+    compile("org.apache.calcite:calcite-core:1.13.0") {
+        // They conflict with embulk-core.
+        exclude group: "org.slf4j", module:"slf4j-api"
+        exclude group: "org.apache.commons", module: "commons-lang3"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-databind"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-annotations"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-core"
+        exclude group: "com.google.guava", module: "guava"
+    }
+
+    testCompile "junit:junit:4.13"
+    testCompile "org.embulk:embulk-core:0.9.23:tests"
+    testCompile "org.embulk:embulk-standards:0.9.23"
+    testCompile "org.embulk:embulk-test:0.9.23"
+
+    // TODO: Remove them.
+    // They are now required because the dependency libraries of them are behind sub ClassLoaders.
+    testCompile "org.embulk:embulk-deps-buffer:0.9.23"
+    testCompile "org.embulk:embulk-deps-config:0.9.23"
+}
+
+embulkPlugin {
+    mainClass = "org.embulk.filter.add_time.AddTimeFilterPlugin"
+    category = "filter"
+    type = "add_time"
+}
```

Comment:
Why `add_time`, not `calcite` ?

Reply:
right. i fixed it on master branch directly before releasing v0.3.0. but thanks for pointing out:-)
https://github.com/muga/embulk-filter-calcite/commit/ff9be40bac8a423902971782d61e774e9fd8eed9

Reply:
Oh, I got it.